dist/autoreviewcomments.user.js

```diff
@@ -1,31 +1,31 @@
-/** @preserve
-// ==UserScript==
-// @name           AutoReviewComments
-// @namespace      benjol
-// @version        1.4.3
-// @description    No more re-typing the same comments over and over!
-// @homepage       https://github.com/Benjol/SE-AutoReviewComments
-// @grant          none
+/** @preserve
+// ==UserScript==
+// @name           AutoReviewComments - Nisarg

```

Comment:
There's really no need for this

Reply:
It's not there in the final commit. I had that initially as I had two scripts on my chrome - the original one, and the one I was making changes to.